### PR TITLE
 arch/armv7-a: change format of abort logs to match with ARMv8-M's

### PR DESCRIPTION
**Pull request by neel-samsung**
This patch changes the log format for data abort and prefetch abort logs to make it as same as fault handler's logs in armv8-m. It also corrects the indentation issues in the change intended files.

armv7-a old abort logs:
arm_dataabort: Data abort. PC: 0e16f0ba DFAR: 00000000 DFSR: 0000000d

armv7-a new abort logs after this commit:
print_dataabort_detail: ######################################################################### print_dataabort_detail: PANIC!!! Data Abort at instruction : 0x0e16f0ba
print_dataabort_detail: PC: 0e16f0ba DFAR: 00000000 DFSR: 0000000d
print_dataabort_detail: #########################################################################

### CODE REVIEW

**Review by samsung-singh — APPROVED** [review]
abort logs format in armv7-a is changed to match the same of armv8-m, this change is good to merge.

File: os/arch/arm/src/armv7-a/arm_prefetchabort.c [hunk 1]
```diff
@@ -65,6 +65,23 @@
...
+ * Name: print_prefetchabort_detail
+ ****************************************************************************/
+
+static inline void print_prefetchabort_detail(uint32_t *regs, uint32_t ifar, uint32_t ifsr)
+{
+	_alert("#########################################################################\n");
+	_alert("PANIC!!! Prefetch Abort at instruction : 0x%08x\n",  regs[REG_PC]);
+	_alert("PC: %08x DFAR: %08x DFSR: %08x\n", regs[REG_PC], ifar, ifsr);
+	_alert("#########################################################################\n\n\n");
+}
```
Comment (sunghan-chang) [inline os/arch/arm/src/armv7-a/arm_prefetchabort.c:83]: almost the same as print_dataabort_detail. 
Can we merge them? Including ARMv8-M's functions

File: os/arch/arm/src/armv7-a/arm_dataabort.c [hunk 2]
```diff
@@ -180,23 +198,25 @@ uint32_t *arm_dataabort(uint32_t *regs, uint32_t dfar, uint32_t dfsr)
...
-  
-  PANIC();
-  regs = (uint32_t *)CURRENT_REGS;
-  CURRENT_REGS = saved_state;
-  return regs; /* To keep the compiler happy */
+
+	PANIC();
+	regs = (uint32_t *)CURRENT_REGS;
+	CURRENT_REGS = saved_state;
+	return regs; /* To keep the compiler happy */
```
Comment (sunghan-chang) [inline os/arch/arm/src/armv7-a/arm_dataabort.c:219]: Formatting should be another commit. Could you split a commit for that?

**Comment by sunghan-chang** [comment]
`os/arch/arm/src/armv7-a: change format of abort logs in armv7-a to match it with fault logs in armv8-m`

The title is too long, how about
arch/arm/armv7-a: change format of abort logs to match with ARMv8-M's

**Comment by neel-samsung** [comment]
> `os/arch/arm/src/armv7-a: change format of abort logs in armv7-a to match it with fault logs in armv8-m`
> 
> The title is too long, how about arch/armv7-a: change format of abort logs to match with ARMv8-M's
Thank you for the suggestion. Yes, we should go with this title.

File: os/arch/arm/src/armv7-a/arm_prefetchabort.c (hunk 1 above)
Comment (sunghan-chang) [reply os/arch/arm/src/armv7-a/arm_prefetchabort.c:83]: Could you let me know your opinion about this?

**Comment by neel-samsung** [comment]
@sunghan-chang ahh.. Sorry, I am not sure why my reply comments are shown pending. I already did replied for all your comments yesterday.
Anyways, API's in ARMV-8M related to this doesn't look similar.

**Review by sunghan-chang — APPROVED** [review]

File: os/arch/arm/src/armv7-a/arm_prefetchabort.c [hunk 3]
```diff
@@ -65,6 +65,23 @@
...
+
+/****************************************************************************
+ * Name: print_prefetchabort_detail
+ ****************************************************************************/
+
+static inline void print_prefetchabort_detail(uint32_t *regs, uint32_t ifar, uint32_t ifsr)
+{
+	_alert("#########################################################################\n");
+	_alert("PANIC!!! Prefetch Abort at instruction : 0x%08x\n",  regs[REG_PC]);
+	_alert("PC: %08x DFAR: %08x DFSR: %08x\n", regs[REG_PC], ifar, ifsr);
```
Comment (neel-samsung) [inline os/arch/arm/src/armv7-a/arm_prefetchabort.c:81]: Instead of ```DFSR```, This should be ```IFSR```? Let me check.

Comment (neel-samsung) [reply os/arch/arm/src/armv7-a/arm_prefetchabort.c]: Done

**Review by sunghan-chang — APPROVED** [review]

**Review by r-prabu — APPROVED** [review]
